<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="currentIndex">
         <number>6</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>253</width>
           <height>394</height>
          </rect>
         </property>
         <attribute name="label">
          <string>QCoreApplication类</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_19">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>返回应用程序名称</string>
          </property>
          <property name="text">
           <string>applicationName()</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_20">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>组织名</string>
          </property>
          <property name="text">
           <string>OrganizationName</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>253</width>
           <height>394</height>
          </rect>
         </property>
         <attribute name="label">
          <string>QFile类</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_21">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>//将文件移除到回收站</string>
          </property>
          <property name="text">
           <string>moveToTrash</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_22">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>same to above</string>
          </property>
          <property name="text">
           <string>exists()</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_23">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>150</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>删除文件</string>
          </property>
          <property name="text">
           <string>remove()</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>253</width>
           <height>394</height>
          </rect>
         </property>
         <attribute name="label">
          <string>QFileInfo类</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>文件信息</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton">
               <property name="toolTip">
                <string>返回文件所在路径，其它不演示</string>
               </property>
               <property name="text">
                <string>path</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="pushButton_6">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="pushButton_7">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="pushButton_8">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="pushButton_9">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="pushButton_10">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QPushButton" name="pushButton_11">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QPushButton" name="pushButton_12">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QPushButton" name="pushButton_13">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QPushButton" name="pushButton_14">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QPushButton" name="pushButton_15">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QPushButton" name="pushButton_16">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QPushButton" name="pushButton_17">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QPushButton" name="pushButton_18">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>253</width>
           <height>394</height>
          </rect>
         </property>
         <attribute name="label">
          <string>QDir类</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_24">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>回驱动器列表</string>
          </property>
          <property name="text">
           <string>drives()</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_25">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>100</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>列出一个目录下的所有子
目录</string>
          </property>
          <property name="text">
           <string>entryList()</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>253</width>
           <height>394</height>
          </rect>
         </property>
         <attribute name="label">
          <string>QTemporaryDir类</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_26">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>在系统临时目录下创建临时目录</string>
          </property>
          <property name="text">
           <string>create()</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_27">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>110</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>在指定目录下创建临时文件夹</string>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_28">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>190</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>在当前目录下创建临时文件夹</string>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>253</width>
           <height>394</height>
          </rect>
         </property>
         <attribute name="label">
          <string>QTemporaryFile类</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_29">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>create()</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_30">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>90</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_31">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>210</y>
            <width>88</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>253</width>
           <height>394</height>
          </rect>
         </property>
         <attribute name="label">
          <string>QFileSystemWatcher类</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_32">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>30</y>
            <width>131</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>addpath()</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_33">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>80</y>
            <width>131</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>removepath()</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_34">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>140</y>
            <width>131</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>directories</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_35">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>210</y>
            <width>121</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>files()</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pbtopenfile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>打开文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtopendir">
           <property name="text">
            <string>打开目录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear">
           <property name="text">
            <string>清除文本框</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>文件</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>信息</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>目录</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="3" column="1">
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
